src/app/pages/organizations/developers/developer/developer-view.jsx

```diff
@@ -142,8 +160,20 @@ function ChplDeveloperView({ dispatch }) {
               developer={developer}
               dispatch={handleProductDispatch}
             />
-          </Box>
+          </>
         )}
+        { (state === 'view' || state === 'editUser')
+          && (
+            <Box>
```

Comment:
Why take away the first box, where there does need to be an enclosing element (like `<>`), but then add a Box around the second thing where an enclosing element shouldn't be needed?